examples/table/table/app.py

```diff
@@ -7,20 +7,31 @@
 # Include some non-string objects to make sure conversion works correctly.
 headings = ["Title", "Year", "Rating", "Genre"]
 bee_movies = [
-    ("The Secret Life of Bees", 2008, 7.3, "Drama"),
-    ("Bee Movie", 2007, 6.1, "Animation, Adventure"),
-    ("Bees", 1998, 6.3, "Horror"),
-    ("The Girl Who Swallowed Bees", 2007, 7.5),  # Missing a genre
-    ("Birds Do It, Bees Do It", 1974, 7.3, "Documentary"),
-    ("Bees: A Life for the Queen", 1998, 8.0, "TV Movie"),
-    ("Bees in Paradise", 1944, 5.4, None),  # None genre
-    ("Keeper of the Bees", 1947, 6.3, "Drama"),
+    ["The Secret Life of Bees", 2008, 7.3, "Drama"],
+    ["Bee Movie", 2007, 6.1, "Animation, Adventure"],
+    ["Bees", 1998, 6.3, "Horror"],
+    ["The Girl Who Swallowed Bees", 2007, 7.5],  # Missing a genre
+    ["Birds Do It, Bees Do It", 1974, 7.3, "Documentary"],
+    ["Bees: A Life for the Queen", 1998, 8.0, "TV Movie"],
+    ["Bees in Paradise", 1944, 5.4, None],  # None genre
+    ["Keeper of the Bees", 1947, 6.3, "Drama"],
```

Comment:
The use of tuple was deliberate here - lists and tuples aren't (or shouldn't be) interchangeable. In this case, position N of the data tuple has a significant meaning. If anything I'd argue the error here is that `headings` should be a tuple, not a list.

Was this actually causing an error?

Reply:
I agree with that general principle, but using tuples made it impossible to add the icons programmatically. I suppose adding them manually wouldn't be so bad.

Reply:
Ah - that makes sense. I guess my concern is that these are examples, and we should endeavour to do the right thing so that novice users don't fall into the "a tuple is just an immutable list" trap.

Two alternate approaches I can think of:
1. Copy the entire row, not just cell 0 when injecting the icon
2. Dynamically generate the bee data as part of the startup method, rather than specifying it as a global constant.

Reply:
OK, I've done it as part of the startup method.